Toolset/libraries/revidelibrary.8.livecodescript

```diff
@@ -1826,57 +1860,83 @@ those paint tools.
 
 **/
 function revIDEPaintTools
-   local tReturnData
-   put "tool" into tReturnData["com.livecode.tool.Select"]["type"]
-   put "select tool" into tReturnData["com.livecode.tool.Select"]["tool"]
-   put "Select Tool - Select a rectangle area of an image" into tReturnData["com.livecode.tool.Select"]["tooltip"]
-   put 7 into tReturnData["com.livecode.tool.Select"]["order"]
-
-   put "tool" into tReturnData["com.livecode.tool.Bucket"]["type"]
-   put "bucket tool" into tReturnData["com.livecode.tool.Bucket"]["tool"]
-   put "Bucket Tool - Fills on outline in an image" into tReturnData["com.livecode.tool.Bucket"]["tooltip"]
-   put 3 into tReturnData["com.livecode.tool.Bucket"]["order"]
-
-   put "tool" into tReturnData["com.livecode.tool.SprayCan"]["type"]
-   put "spray tool" into tReturnData["com.livecode.tool.SprayCan"]["tool"]
-   put "Spray Can Tool - Draws airbrush strokes on an image" into tReturnData["com.livecode.tool.SprayCan"]["tooltip"]
-   put 4 into tReturnData["com.livecode.tool.SprayCan"]["order"]
-
-   put "tool" into tReturnData["com.livecode.tool.Eraser"]["type"]
-   put "eraser tool" into tReturnData["com.livecode.tool.Eraser"]["tool"]
-   put "Eraser Tool - Erases an area in an image" into tReturnData["com.livecode.tool.Eraser"]["tooltip"]
-   put 8 into tReturnData["com.livecode.tool.Eraser"]["order"]
-
-   put "tool" into tReturnData["com.livecode.tool.Polygon"]["type"]
-   put "polygon tool" into tReturnData["com.livecode.tool.Polygon"]["tool"]
-   put "Polygon Tool - Click to set each point on the polygon, double click to close the polygon" into tReturnData["com.livecode.tool.Polygon"]["tooltip"]
-   put 5 into tReturnData["com.livecode.tool.Polygon"]["order"]
-
-   put "tool" into tReturnData["com.livecode.tool.Curve"]["type"]
-   put "curve tool" into tReturnData["com.livecode.tool.Curve"]["tool"]
-   put "Curve Tool - Draws a curved line on an image" into tReturnData["com.livecode.tool.Curve"]["tooltip"]
-   put 6 into tReturnData["com.livecode.tool.Curve"]["order"]
-
-   put "tool" into tReturnData["com.livecode.tool.Pencil"]["type"]
-   put "pencil tool" into tReturnData["com.livecode.tool.Pencil"]["tool"]
-   put "Pencil Tool - Draws a freehand line on an image" into tReturnData["com.livecode.tool.Pencil"]["tooltip"]
-   put 1 into tReturnData["com.livecode.tool.Pencil"]["order"]
-
-   put "tool" into tReturnData["com.livecode.tool.Brush"]["type"]
-   put "brush tool" into tReturnData["com.livecode.tool.Brush"]["tool"]
-   put "Brush Tool - Draws freehand brush strokes on an image" into tReturnData["com.livecode.tool.Brush"]["tooltip"]
-   put 2 into tReturnData["com.livecode.tool.Brush"]["order"]
-
-   return tReturnData
+   local tDataA
+   
+   local tTooltip, tName, tCount
+   
+   add 1 to tCount
+   put "com.livecode.tool.Pencil" into tName
+   put "Pencil Tool - Draws a freehand line on an image" into tTooltip
```

Comment:
@livecodeali What do you think about standardising on imperative voice for these tooltips? E.g. "Draw a freehand line on an image", "Fill an outline in an image", "Select a rectangle area of an image," and so on?